### DISCUSSION TITLE
Invalid Syntax Error

### DISCUSSION THREAD

**Discussion by macuser2** [UltimaHoarder/UltimaScraper#969] [Q&A]
I am on Mac Big Sur 11.2.3

python 3 is set as my default version.

I get a warning code when I use `pip3 install -r requirements.txt`
"WARNING: deepdiff 5.2.3 does not provide the extra 'murmur'"
*All other requirements are satisfied.*

Also, when I try `python start_ofd.py`

I get:

"Traceback (most recent call last):
  File "/Users/User/Downloads/OnlyFans-6.4.2/start_ofd.py", line 2, in <module>
    import tests.main_test as main_test
  File "/Users/User/Downloads/OnlyFans-6.4.2/tests/main_test.py", line 12
    if python_version < 3.9.2:
                           ^
SyntaxError: invalid syntax"

Anyone got any advice?

**Comment by UltimaHoarder** [comment]
Use Python 3.9

**Reply by macuser2** [reply]
Currently my python version is 3.9.2, so do I need to downgrade?

**Reply by UltimaHoarder** [reply]
No, your system most likely has another version of Python installed.

**Reply by macuser2** [reply]
So I got a little further. Deleted and redownloaded and used the latest file 6.4.4. After I run the `start python_ofd.py` command, I get 
"The .settings\config.json file has been updated. Fill in whatever you need to fill in and then press enter when done."

So I go into my file folder to find the settings folder but it's not there still.

**Reply by macuser2** [reply]
Problem solved. I did not have the "view hidden files" enabled on Mac.

**Reply by kevinsheeks** [reply]
did u ever figure out the config.json issue im having problems myself

**Reply by macuser2** [reply]
I had to run the `python start_ofd` twice. Once gets you the .settings folder. Second time gets you the .profiles folder.

It worked fine a few days ago. Now today I'm having issues again.

**Reply by kevinsheeks** [reply]
yeah i figured out its n the profiles folder now lol
